sdk/cognitiveservices/Language.TextAnalytics/src/README.md

```diff
@@ -1,11 +1,10 @@
-# Microsoft Cognitive Services Language SDK for .NET
+# Microsoft Azure Text Analytics SDK library for .NET
  ----
+**Note:** This SDK targets the V2.1 version of the Text Analytics service. For the new set of libraries that support newer service versions please see the Text Analytics Azure SDK under the path: https://github.com/Azure/azure-sdk-for-net/tree/master/sdk/textanalytics/Azure.AI.TextAnalytics
```

Comment:
Is there any public documentation on Track 1 vs. Track 2 libraries?  If so, it might be nice to call out that this is a track 1 library and the new library is track 2.

Reply:
Might also be nice to briefly call out that you'll find in the README instructions for installing the new Azure.AI.TextAnalytics package and samples showing its use?  (Just thinking because that README contains so much more than this README.)

Reply:
Nit:

```suggestion
**Note:** This SDK targets the V2.1 version of the Text Analytics service. For the new set of libraries that support newer service versions please see the Text Analytics Azure SDK under the path: https://github.com/Azure/azure-sdk-for-net/tree/master/sdk/textanalytics/Azure.AI.TextAnalytics .
```

Reply:
I couldn't find one that will help, but added a reference to our guidelines like we do here: https://github.com/Azure/azure-sdk-for-net#client

Reply:
section updated